wolfcrypt/test/test.c

```diff
@@ -10178,7 +10178,7 @@ static int ecc_sig_test(WC_RNG* rng, ecc_key* key)
     int     ret;
     word32  sigSz;
     int     size;
-    byte    out[65];
+    byte    out[75];
```

Comment:
Is a good idea to remove the magic number. Am thinking maybe ECC_MAX_SIG_SIZE is a better fit though.